config/watcher.go

```diff
@@ -54,12 +54,17 @@ func newCloudWatcher(ctx context.Context, config *Config, logger logging.Logger)
 
 	nextCheckForNewCert := time.Now().Add(checkForNewCertInterval)
 
-	ticker := time.NewTicker(config.Cloud.RefreshInterval)
```

Comment:
wait a second, were we even using this before?

Reply:
nope